database/migrations/2020_09_18_172858_add_mission_field_for_task.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddMissionFieldForTask extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('tasks', function (Blueprint $table) {
+            $table->text('mission_uid')->nullable()->after('uid');
```

Comment:
```suggestion
            $table->uuid('mission_uid')->nullable()->after('uid');
```

會不會比較好？另外是 `mission_uid` 是可以 nullable?

Reply:
nullable 是我想說可以更換任務

Reply:
更換任務的話應該還是有任務？或是說會有情境是 `mission_uid` 是 null 的情況？